docs/security_vulnerability_disclosure/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Guidelines for Security Vulnerability Disclosure for Library Authors on the BEAM
+layout: page
+next:
+  url: target_audience
+  title: Target Audience
+---
+
+In the dynamic and rapidly evolving landscape of software development, ensuring
+the security of libraries and frameworks is paramount. The BEAM ecosystem,
+comprising Erlang and Elixir, is renowned for its robustness, scalability, and
+fault-tolerance. However, like any other technology stack, it is not immune to
+security vulnerabilities. As library authors play a crucial role in maintaining
+the integrity and security of the BEAM ecosystem, it becomes imperative to
+establish clear guidelines for disclosing and addressing security
+vulnerabilities.
+
+This document serves as a comprehensive guide for library authors within the
+BEAM community, outlining the processes and best practices for handling security
+vulnerabilities effectively. By following these guidelines, library authors can
+contribute to enhancing the overall security posture of the BEAM ecosystem while
+fostering transparency and collaboration among developers, security researchers,
+and end-users.
+
+The objectives of this document are threefold:
+
+* **Clarifying Responsibilities:** Library authors need to understand their
+  responsibilities concerning the discovery, disclosure, and mitigation of
+  security vulnerabilities in their codebases. Clear delineation of roles and
+  expectations ensures a coordinated and timely response to security incidents.
+* **Establishing Reporting Mechanisms:** Effective communication channels are
+  vital for reporting security vulnerabilities. This document outlines the
+  preferred methods and contacts for reporting security issues, facilitating
+  prompt assessment and remediation.
+* **Implementing Remediation Procedures:** Upon receiving reports of security
+  vulnerabilities, library authors must follow structured procedures for
+  triaging, validating, and addressing the reported issues. Timely patches or
+  updates should be developed and communicated to users to mitigate potential
+  risks.
+
+By adhering to these guidelines, library authors can demonstrate their
+commitment to maintaining a secure and resilient BEAM ecosystem, earning the
+trust and confidence of their users and stakeholders. Together, we can create a
+safer environment for building and deploying Erlang and Elixir applications,
+safeguarding critical systems and sensitive data against emerging threats.
+
+## Contents
+
+* [Target Audience](target_audience)
+* [Vulnerability Definition](vulnerability_definition)
+* [User Tooling](user_tooling)
+* [Process](process)
+* [GitHub Disclosure](github_disclosure)
+
+The ErlEF Security Working Group ([security@erlef.org](mailto:security@erlef.org))
+offers to connect individuals with experts who can assist in coordinating and
+disclosing vulnerabilities within the BEAM ecosystem. While the WG doesn't
```

Comment:
Negative contractions are fine in informal English, AFAIK.
My personal opinion is that we should avoid them, because it looks more formal without negative contractions

Reply:
@kikofernandez How would you word this instead?

I want to make clear, that we're willing to offer support. But we don't want people to send us vulnerabilities to the mailing list.

Reply:
This could be "does not" instead of "doesn't".